### PR TITLE
Adding post on Helm Hub

### PR DESCRIPTION
**Pull request by mattfarina**
DO NOT MERGE UNTIL THE ANNOUNCEMENT IS DONE

This does two things...

1. Adds a blog post announcing the Helm Hub
2. Updates the chart discovery on the homepage to point to the hub

screenshot_2018-12-11 introducing the helm hub

### CODE REVIEW

File: helm.sh/_posts/2018-12-11-helm-hub.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+With this in mind, we are delighted to announce the launch of the [Helm Hub](https://hub.helm.sh). This hub provides a means for you to find charts hosted in many distributed repositories hosted by numerous people and organizations. <!--more-->
+
+![Helm Hub](/assets/images/blog/helm-hub.png)
+
+Helm repositories can be hosted in many ways including as GitHub or Gitlab pages, in object storage, using [Chartmuseum](https://github.com/helm/chartmuseum), and via a service provider.
+
+If you have a chart repository you would like listed please head over to the [hub repository on GitHub](https://github.com/helm/hub) and follow the directions. The process is as simple as a pull request.
+
+The Helm Hub is powered by [Monocular](https://github.com/helm/monocular), one of the projects that has long been a part of Helm. This was originally crafted by Bitnami and Deis, now part of Microsoft. As the Helm Hub grows in complexity, Monocular will need to grow in its ability to handle many repositories and charts. This is an area that UI and UX designers looking to be part of the Helm and [CNCF](https://cncf.io) community can contribute and make a difference.
```
Comment (scottrigby) [inline helm.sh/_posts/2018-12-11-helm-hub.md]: Quick read, this is all great Matt! A few suggestions:

This is an area *where*...
or
...can contribute *to* and make a difference

File: helm.sh/_posts/2018-12-11-helm-hub.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+We look forward to the Helm Hub ushering in a new phase of chart development and sharing.
```
Comment (scottrigby) [inline helm.sh/_posts/2018-12-11-helm-hub.md]: No newline at EOF

File: helm.sh/_posts/2018-12-11-helm-hub.md (hunk 1 above)
Comment (mattfarina) [reply helm.sh/_posts/2018-12-11-helm-hub.md]: fixed

File: helm.sh/_posts/2018-12-11-helm-hub.md (hunk 2 above)
Comment (mattfarina) [reply helm.sh/_posts/2018-12-11-helm-hub.md]: fixed

**Comment by unguiculus** [comment]
LGTM

**Comment by rimusz** [comment]
👍

**Pull request merged**